.eslintrc.js

```diff
@@ -4,16 +4,31 @@ module.exports = {
     node: true
   },
   extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:vue/base',
     'plugin:vue/essential',
+    // 'plugin:vue/recommended',
+    // 'plugin:vue/strongly-recommended',
+    'plugin:vue/vue3-essential',
+    // 'plugin:vue/vue3-recommended',
+    // 'plugin:vue/vue3-strongly-recommended',
+    'plugin:vuetify/base',
+    'plugin:vuetify/recommended',
     '@vue/standard',
     '@vue/typescript'
   ],
   rules: {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'max-len': ['warn', { code: 120, ignoreRegExpLiterals: true }],
+    // 'vue/attribute-hyphenation': 'off',
     // 'vue/no-deprecated-filter': 'warn',
+    'vue/no-deprecated-slot-scope-attribute': 'warn',
     // 'vue/no-deprecated-v-on-native-modifier': 'warn',
+    'vue/no-v-for-template-key-on-child': 'warn',
```

Comment:
Similar to above. Now we get this:

![image](https://user-images.githubusercontent.com/10002889/233749754-4abb7677-f970-49ae-8a03-94b53c9316bb.png)